### PR TITLE
[fixes #6515] - Allow custom subject and screen size for scheduled email

### LINKED ISSUE #6515: Feature: Allow custom screen size for scheduled email slices and dashboards
Make sure these boxes are checked before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

### Steps to reproduce

@mahendra: Since the PR merged, please can you add custom screen size option and able to configure via UI for scheduled email slices and dashboard?

### PR DESCRIPTION
**Pull request by mahendra**

### CODE REVIEW

File: superset/tasks/schedules.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
 # Time in seconds, we will wait for the page to load and render
 PAGE_RENDER_WAIT = 30
 
+DATETIME_FORMAT = '%Y/%m/%d/%H/%M/%S'
```
Comment (hughhhh) [inline superset/tasks/schedules.py]: nit: This should be in the `config.py` and allow users to set this accordingly

File: superset/migrations/versions/721b47faf907_custom_fields_for_email_reports.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+"""Custom fields for email reports
```
Comment (bolkedebruin) [inline superset/migrations/versions/721b47faf907_custom_fields_for_email_reports.py]: Add ASF license header please

File: superset/views/schedules.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # pylint: disable=C,R,W
```
Comment (bolkedebruin) [inline superset/views/schedules.py]: Add ASF license header please

**Review by bolkedebruin — CHANGES_REQUESTED** [review]

File: superset/tasks/schedules.py (hunk 1 above)
Comment (mahendra) [reply superset/tasks/schedules.py]: @hughhhh thank you for the comment. This is not meant for user configuration at all. This is just to define an interface for passing the task's run time as an argument to a celery task.

File: superset/migrations/versions/721b47faf907_custom_fields_for_email_reports.py (hunk 2 above)
Comment (mahendra) [reply superset/migrations/versions/721b47faf907_custom_fields_for_email_reports.py]: oops. done.

File: superset/views/schedules.py (hunk 3 above)
Comment (mahendra) [reply superset/views/schedules.py]: looks like it was added upstream

**Comment by mmuru** [comment]
@mahendra: I verified the changes and all looks good. Can you rebase and update Alembic revision for the migration script?

@mistercrunch: Can we merge this PR into master once the migration db script revision updated?

**Pull request closed without merging**